### ISSUE TITLE
issue installing on BitCurator

### ISSUE THREAD

**Issue by ccmn98** [tw4l/brunnhilde-gui#5]
I tried to install the GUI on the BitCurator OVA file in Virtual box but in the terminal I am given the message, "tee: invalid option -- 'e' ." 

What changes can I do to successfully install the GUI?

Thanks

**Comment by tw4l** [comment]
Thanks @ccmn98 ! Can you share with me the command you used to try to install in BitCurator, and also the version of BitCurator you're using?

**Comment by gwiedeman** [comment]
I had this same issue on bitcurator 4.4.3, and found some super simple fixes.

I installed it yesterday near COB and got an error (which of course I forget). It did make `/usr/share/applications/BrunnhildeGUI.desktop`  (although it didn't appear in the application menu as I expected). So I  _was_ able to run it with the path listed in that file: `/use/share/brunnhilde-gui/launch` and do what I needed to do.

I returned today and ran the install again to see what was up, and I get this error:
```
greg@bcadmin-OptiPlex-9020:~/brunnhilde-gui$ sudo ./install
Requirement already satisfied: pyqt5 in /usr/lib/python3/dist-packages (5.15.6)
Requirement already satisfied: PyQt5-sip<13,>=12.8 in /usr/lib/python3/dist-packages (from pyqt5) (12.9.1)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
mv: cannot stat 'design.py': No such file or directory
mv: cannot stat 'design.ui': No such file or directory
mv: cannot stat 'icon.png': No such file or directory
mv: cannot stat 'launch': No such file or directory
mv: cannot stat 'main.py': No such file or directory
[Desktop Entry]
Type=Application
Name=Brunnhilde GUI
Exec=/usr/share/brunnhilde-gui/launch
Icon=/usr/share/brunnhilde-gui/icon.png
tee: invalid option -- 'e'
Try 'tee --help' for more information.
chmod: cannot access '/usr/share/brunnhilde-gui/launch': No such file or directory
```

`/usr/share/brunnhilde-gui` exists, but is empty, as the files in the repo got moved the first time with `mv` so they're no longer in my  current directory. Replacing `mv` with `cp` should make this re-runable.

Re-cloning the repo and running install again I get what I suspect is the original error:
```
greg@bcadmin-OptiPlex-9020:~$ git clone https://github.com/tw4l/brunnhilde-gui
Cloning into 'brunnhilde-gui'...
remote: Enumerating objects: 310, done.
remote: Counting objects: 100% (16/16), done.
remote: Compressing objects: 100% (9/9), done.
remote: Total 310 (delta 8), reused 13 (delta 7), pack-reused 294
Receiving objects: 100% (310/310), 239.46 KiB | 6.47 MiB/s, done.
Resolving deltas: 100% (182/182), done.
greg@bcadmin-OptiPlex-9020:~$ cd brunnhilde-gui/
greg@bcadmin-OptiPlex-9020:~/brunnhilde-gui$ sudo ./install
Requirement already satisfied: pyqt5 in /usr/lib/python3/dist-packages (5.15.6)
Requirement already satisfied: PyQt5-sip<13,>=12.8 in /usr/lib/python3/dist-packages (from pyqt5) (12.9.1)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
[Desktop Entry]
Type=Application
Name=Brunnhilde GUI
Exec=/usr/share/brunnhilde-gui/launch
Icon=/usr/share/brunnhilde-gui/icon.png
tee: invalid option -- 'e'
Try 'tee --help' for more information.
```

Looking at `install.sh`, this is just because its [missing a dash](https://github.com/tw4l/brunnhilde-gui/blob/main/install#L30) (`-`).

It still doesn't show up in my applications menu, because `chown` isn't working right as I am not `bcadmin`. It should be pretty simple to set this up for multiple users. I'll put in a PR for these fixes/improvements.

**Comment by tw4l** [comment]
Fixed in #6 thanks to @gwiedeman ! Just released v2.0.1 with the fix :)
